changelogs/fragments/gha-organize-workflows.yaml

```diff
@@ -0,0 +1,3 @@
+---
+trivial:
+  - organize the workflows and pin ansible-lint to 6.21.x.
```

Comment:
I think since we're changing the meta/runtime.yml to require ansible 2.14+ we should mark this PR as a breaking change.